python/dgl/nn/pytorch/gt/spatial_encoder.py

```diff
@@ -87,51 +90,50 @@ class SpatialEncoder3d(nn.Module):
     `One Transformer Can Understand Both 2D & 3D Molecular Data
     <https://arxiv.org/pdf/2210.01765.pdf>`__
 
-    This module encodes pair-wise relation between atom pair :math:`(i,j)` in
+    This module encodes pair-wise relation between node pair :math:`(i,j)` in
     the 3D geometric space, according to the Gaussian Basis Kernel function:
 
-    :math:`\psi _{(i,j)} ^k = -\frac{1}{\sqrt{2\pi} \lvert \sigma^k \rvert}
+    :math:`\psi _{(i,j)} ^k = \frac{1}{\sqrt{2\pi} \lvert \sigma^k \rvert}
     \exp{\left ( -\frac{1}{2} \left( \frac{\gamma_{(i,j)} \lvert \lvert r_i -
     r_j \rvert \rvert + \beta_{(i,j)} - \mu^k}{\lvert \sigma^k \rvert} \right)
     ^2 \right)}，k=1,...,K,`
 
-    where :math:`K` is the number of Gaussian Basis kernels.
-    :math:`r_i` is the Cartesian coordinate of atom :math:`i`.
-    :math:`\gamma_{(i,j)}, \beta_{(i,j)}` are learnable scaling factors of
-    the Gaussian Basis kernels.
+    where :math:`K` is the number of Gaussian Basis kernels. :math:`r_i` is the
+    Cartesian coordinate of node :math:`i`.
+    :math:`\gamma_{(i,j)}, \beta_{(i,j)}` are learnable scaling factors and
+    biases determined by node types. :math:`\mu^k, \sigma^k` are learnable
+    centers and standard deviations of the Gaussian Basis kernels.
 
     Parameters
     ----------
     num_kernels : int
-        Number of Gaussian Basis Kernels to be applied.
-        Each Gaussian Basis Kernel contains a learnable kernel center
-        and a learnable scaling factor.
+        Number of Gaussian Basis Kernels to be applied. Each Gaussian Basis
+        Kernel contains a learnable kernel center and a learnable standard
+        deviation.
     num_heads : int, optional
         Number of attention heads if multi-head attention mechanism is applied.
         Default : 1.
     max_node_type : int, optional
-        Maximum number of node types. Default : 1.
+        Maximum number of node types. Each node type has a corresponding
+        learnable scaling factor and a bias. Default : 100.
 
     Examples
     --------
     >>> import torch as th
     >>> import dgl
     >>> from dgl.nn import SpatialEncoder3d
 
-    >>> u = th.tensor([0, 0, 0, 1, 1, 2, 3, 3])
-    >>> v = th.tensor([1, 2, 3, 0, 3, 0, 0, 1])
-    >>> g = dgl.graph((u, v))
-    >>> coordinate = th.rand(4, 3)
-    >>> node_type = th.tensor([1, 0, 2, 1])
+    >>> coordinate = th.rand(1, 4, 3)
+    >>> node_type = th.tensor([[1, 0, 2, 1]])
```

Comment:
It might be better to use the same example for doc and unit tests.

Reply:
added